IdentityCore/src/requests/sdk/MSIDBrokerResponseHandler.m

```diff
@@ -122,7 +122,21 @@ - (MSIDTokenResult *)handleBrokerResponseWithURL:(NSURL *)response error:(NSErro
         if (error) *error = brokerError;
         return nil;
     }
-
+    
+    NSString *applicationToken = brokerResponse.applicationToken;
+    
+    if (![NSString msidIsStringNilOrBlank:applicationToken])
+    {
+        NSError *appTokenError = nil;
+        BOOL saveAppToken = [brokerKeyProvider saveApplicationToken:applicationToken forClientId:brokerResponse.clientId error:&appTokenError];
+        
+        if (!saveAppToken && appTokenError)
```

Comment:
nit: no need to check for both. Here it's enough to check for saveAppToken result only.

Reply:
Addressed.